cmd/main.go

```diff
@@ -67,13 +66,7 @@ func findGoModulePath(forceModules bool) (string, error) {
 // findCurrentRepo attempts to determine the current repository
 // though a combination of go/packages and `go mod` commands/tricks.
 func findCurrentRepo() (string, error) {
-	// easiest case: project file already exists
-	projFile, err := scaffold.LoadProjectFile("PROJECT")
-	if err == nil {
-		return projFile.Repo, nil
-	}
```

Comment:
We were failing if the project was configured, so this step will never succeed or we will fail due to the existance of the configuration file.